packages/tabs/src/Tabs.tsx

```diff
@@ -204,17 +207,18 @@ function Tabs({
             <TabTitle
               {...filteredRest}
               key={index}
+              containerNode={containerNode}
               ariaControl={`tab-${index}`}
               disabled={disabled}
               selected={selected}
               index={index}
               as={as}
               darkMode={darkMode}
+              onKeyDown={handleArrowKeyPress}
```

Comment:
it seems more logical to have this adjacent to the `onClick` prop

Reply:
Was just grouping multiline props at the bottom